projects/storefrontstyles/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@spartacus/styles",
-  "version": "4.1.0-next.0",
+  "version": "4.1.0",
```

Comment:
I'm assuming this was updated due to the fact that you ran verdaccio, which automatically bumps things like this. 

This is should not have been pushed . So all references in this file that had there version updated needs to be changed back to the default state

in `creating-lib.md` under `Testing Schematics`
it should have stated to not push the changes done under the tests. We're sorry for this, but thank you for making me aware of this issue!